### ISSUE TITLE
AccessToken

### ISSUE THREAD

**Issue by pjjjv** [Janamou/firebase3-dart#35]
I can not access response.credential.accessToken where response is a UserCredential. Can you please make sure that all the js object properties and methods are mapped to dart consistently?

**Comment by pjjjv** [comment]
See result.credential.accessToken on https://firebase.google.com/docs/auth/web/google-signin

**Comment by Janamou** [comment]
Hi @pjjjv,
thanks for the info. The `accessToken` was not explicitly mentioned in the API documentation for the `AuthCredential.` I added it with `secret`, which is used with `TwitterAuthProvider`.

You can try it by using the library version in this repository. Update pubspec.yaml:

```
dependencies:
  firebase3:
    git: https://github.com/Janamou/firebase3-dart
```

I will release it to the pub.dartlang.org later with more updates probably.

Does it work now? Thx

**Comment by Janamou** [comment]
Fixed in 2d6ae8fab3a8fd2be0bdff1db3c9192948af57c4
